lumispy/signals/common_luminescence.py

```diff
@@ -214,3 +224,300 @@ def normalize(self, pos=float("nan"), element_wise=False, inplace=False):
                 s.metadata.Signal.quantity = "Normalized intensity"
         if not inplace:
             return s
+
+    def _reset_variance_linear_model(self):
```

Comment:
Am I understanding correctly that this is simply moved around and doesn't need reviewed of the code itself?

Reply:
Yes, all the code pertaining to signal axis conversion is moved to the `CommonLuminescence` class so that it is available also to `LumiTransientSpectrum`. That was the origin of this PR and I probably should have merged that moving part in before doing the other changes to make it clearer.